proto/evmos/vesting/v1/events.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+package evmos.vesting.v1;
+
+option go_package = "github.com/evmos/evmos/v10/x/vesting/types";
+
+// EventCreateClawbackVestingAccount defines the event type
+// for creating a clawback vesting account
+message EventCreateClawbackVestingAccount {
+  // sender is the address of the sender
+  string sender = 1;
+  // coins to be vested
+  string coins = 2;
+  // start_time is the time when the coins start to vest
+  string start_time = 3;
```

Comment:
are we using string for time.Duration?

Reply:
@fedekunze from the previous version of the event emitting it seems it was being converted to string.

Reply:
could you check if the SDK module also uses strings for everything? It doesn't seem beneficial to use proto if everything is still encoded as string